src/indexer/chains/stacks.rs

```diff
@@ -241,90 +238,136 @@ pub fn get_value_description(raw_value: &str) -> String {
     value
 }
 
-pub fn get_tx_description(raw_tx: &str) -> String {
+pub fn get_tx_description(
+    raw_tx: &str,
+) -> Result<
+    (
+        String,
+        StacksTransaction,
+        StacksTransactionKind,
+        u64,
+        String,
+        Option<String>,
+    ),
```

Comment:
Might be nice to package this up into a struct, or add comments describing what they are.

Reply:
hmmm I'll probably go with comments, as this struct would be dereferenced and destructured a few lines later, for feeding another bigger structure.

Reply:
Addressed with latest